app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,317 @@
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_announce" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_1"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="7dp"
+            android:adjustViewBounds="true"
+            android:src="@drawable/image_1"
+            app:layout_constraintEnd_toStartOf="@+id/IV_cast_image_2"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_cast" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_2"
+            app:layout_constraintEnd_toStartOf="@+id/IV_cast_image_3"
+            app:layout_constraintStart_toEndOf="@+id/IV_cast_image_1"
+            app:layout_constraintTop_toTopOf="@id/IV_cast_image_1" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_3"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_3"
+            app:layout_constraintEnd_toStartOf="@+id/IV_cast_image_4"
+            app:layout_constraintStart_toEndOf="@+id/IV_cast_image_2"
+            app:layout_constraintTop_toTopOf="@id/IV_cast_image_2" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_4"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_4"
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toEndOf="@+id/IV_cast_image_3"
+            app:layout_constraintTop_toTopOf="@id/IV_cast_image_3" />
+
+        <TextView
+            android:id="@+id/TV_cast_imagine_text_1"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="6dp"
+            android:adjustViewBounds="true"
+            android:gravity="start"
+            android:text="@string/robert_downey_jr"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toStartOf="@+id/TV_cast_imagine_text_2"
```

Comment:
please relate this and all actors names
app:layout_constraintStart_toStartOf to same actor image above
app:layout_constraintEnd_toEndOf to same actor image above